### DISCUSSION TITLE
Recommendation on using Linkerd edge vs Linkerd stable version in production

### DISCUSSION THREAD

**Discussion by shahriak** [linkerd/linkerd2#5176] [Q&A] [answered]
Hello all,

We are planning to use Linkerd edge versions in production, however, since the versions are rolled out frequently, do older version also get deprecated frequently? I have not found anything regarding deprecation in the official site. If someone shed some light into this, it would be very helpful!

**Comment by grampelberg** [comment, marked as answer]
@shahriak edge versions aren't supported and are meant to be unstable off master at any point in time. There are no guarantees and breakage happens on a semi-regular basis. As we're in the run-up to stable 2.9 right now, there won't be many differences between the current edges and the released 2.9. If you are concerned about support and deprecation cycles, I would recommend upgrading to 2.9 when it comes out and then using edge releases to test new functionality outside of production.
